partner_communication_switzerland/data/sponsorship_communications_cron.xml

```diff
@@ -1,44 +1,76 @@
-            <field name="name">Generate tax receipts</field>
-            <field name="interval_number">12</field>
-            <field name="interval_type">months</field>
-            <field name="numbercall">-1</field>
-            <field name="model_id" ref="model_res_partner"/>
-            <field name="state">code</field>
-            <field name="code">model.generate_tax_receipts()</field>
-            <field name="active" eval="False"/>
-        </record>
+        <template>
+            <!-- Reccuring CRONs that cannot be replaced by automated actions -->
+            <record id="sponsorship_daily_communication_cron" model="ir.cron">
+                <field name="name">Send daily sponsorship communication</field>
+                <field name="interval_number">1</field>
+                <field name="interval_type">days</field>
+                <field name="numbercall">-1</field>
+                <field name="model_id" ref="model_recurring_contract"/>
+                <field name="state">code</field>
+                <field name="code">model.send_daily_communication()</field>
+                <field name="active" eval="False"/>
+            </record>
+            <record id="sponsorship_reminders_cron" model="ir.cron">
+                <field name="name">Send sponsorship reminders</field>
+                <field name="interval_number">1</field>
+                <field name="interval_type">months</field>
+                <field name="numbercall">-1</field>
+                <field name="model_id" ref="model_recurring_contract"/>
+                <field name="state">code</field>
+                <field name="code">model.send_sponsorship_reminders()</field>
+                <field name="active" eval="False"/>
+            </record>
+            <record id="yearly_tax_receipt_cron" model="ir.cron">
+                <field name="name">Generate tax receipts</field>
+                <field name="interval_number">12</field>
+                <field name="interval_type">months</field>
+                <field name="numbercall">-1</field>
+                <field name="model_id" ref="model_res_partner"/>
+                <field name="state">code</field>
+                <field name="code">model.generate_tax_receipts()</field>
+                <field name="active" eval="False"/>
+            </record>
+
+            <!-- Sponsorship anniversaries (1, 3, 5, 10 and 15 years) -->
+            <t t-foreach="[1,3,5,10,15]" t-as="year">
+                <t t-set="record_id" t-value="'sponsorship_anniversary_%d_communication' % year"/>
+                <t t-set="record_name" t-value="'Send sponsorship anniversary %d year %s communication' % (year, 's' if year != 1 else '')"/>
+                <t t-set="config_id" t-value="'planned_anniversary_%d' % year"/>
+                <t t-set="date_range" t-value="12 * year"/>
+                <t t-set="min_date" t-value="Datetime.now() + relativedelta(years=years)"/>
```

Comment:
I think it's the opposite, otherwise we will have to wait a few years until we have new communications generated.
```suggestion
                <t t-set="min_date" t-value="Datetime.now() - relativedelta(years=years)"/>
```